staging/src/k8s.io/legacy-cloud-providers/azure/clients/deploymentclient/azure_deploymentclient.go

```diff
@@ -60,11 +60,11 @@ func New(config *azclients.ClientConfig) *Client {
 	armClient := armclient.New(authorizer, baseURI, config.UserAgent, APIVersion, config.Location, config.Backoff)
 	rateLimiterReader, rateLimiterWriter := azclients.NewRateLimiter(config.RateLimitConfig)
 
-	klog.V(2).Infof("Azure DeploymentClient (read ops) using rate limit config: QPS=%g, bucket=%d",
-		config.RateLimitConfig.CloudProviderRateLimitQPS,
+	klog.V(2).InfoS("Azure DeploymentClient (read ops) using rate limit config", "QPS=",
```

Comment:
```suggestion
	klog.V(2).InfoS("Azure DeploymentClient (read ops) using rate limit config", "QPS",
```

Reply:
Small nit: it might also read better to put the key and value in the same line:
```go
klog.V(2).InfoS("Azure DeploymentClient (read ops) using rate limit config"
    "qps", config.RateLimitConfig.CloudProviderRateLimitQPS,
    "bucket", config.RateLimitConfig.CloudProviderRateLimitBucket)
```